tensorflow/compiler/jit/xla_gpu_device.cc

```diff
@@ -79,8 +82,8 @@ Status XlaGpuDeviceFactory::CreateDevices(
       gpu_ids.insert(platform_gpu_id);
     }
   }
-  for (int i = 0; i < num_visible_devices; ++i) {
-    if (gpu_ids.count(i) == 0) continue;
+  for (const auto i : gpu_ids) {
```

Comment:
I like `auto` as much as the next guy, but here could we just write `for (int i : gpu_ids)`?  That's shorter and more explicit.